Feature/promotion-contact

abdalla-elmahdy opened the pull request:
- Made changes to markup of contact and promotion sections for semantic purposes
- Added styles to promotion and contact sections

abdalla-elmahdy committed:
Semantic fixes for promotion and contact

- Separated promotion and contact us into their own sections
- Removed unnecessary div containers
- lines 149 and 156: swapped h3 with p for semantic purposes

abdalla-elmahdy committed:
Promotion and contact styles

- Added a flex container around promotion and contact section
- Added and adjusted background images to both sections
- Centered content of both sections vertically and horizontally
- Added text styles

muubar commented on dist/assests/css/style.css:
```diff
@@ -78,11 +78,44 @@ ul {
+  flex-flow: row wrap;
+}
+
+.promotion-contact-flexwrapper section {
```
try to avoid element selectors in CSS whenever possible, CSS classes should be your main way of styling.
recommended reads:
https://css-tricks.com/strategies-keeping-css-specificity-low/
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css

abdalla-elmahdy replied:
Got it! will fix it right away.

muubar commented on dist/assests/css/style.css:
```diff
@@ -78,11 +78,44 @@ ul {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 55vh;
```
good job using relative units!

muubar commented on dist/index.html:
```diff
@@ -142,22 +142,21 @@ <h3>Sarra Miller</h3>
-  </section>
+  <div class="promotion-contact-flexwrapper">
+    <!-- Promotions Section -->
+    <section class="promotions-section">
```
good use of semantic HTML tags, good job!

muubar commented on dist/index.html:
```diff
@@ -142,22 +142,21 @@ <h3>Sarra Miller</h3>
+    <section class="contact-section">
+      <h2>call us and try</h2>
+      <p>957 547 4545</p>
+      <button class="btn btn-primary">get in touch</button>
```
when naming your CSS classes, try to not use generic names like `btn` unless the styling is truly generic, otherwise try to be more specific in the naming, an example from your code would be : `promotion-contact-flexwrapper`, you didn't just call it `flexwrapper`

abdalla-elmahdy replied:
I actually used the generic styles for buttons made by @ahmedsamir96m, It's the same button across the page.

Nilegfx reviewed (changes requested):
Please move your custom styles beneath normalize.css styles in the css

abdalla-elmahdy committed:
Lowered the specificity of selectors 

- Added new classes to use for styling
- Swapped high specificity selectors with lower specificity class selectors

abdalla-elmahdy committed:
Separated promotion and contact styles into their own CSS file

- Created new file and added the styles to it
- Removed the styles from style.css
- Added link for the new CSS file in index.html

abdalla-elmahdy committed:
Resolved conflicts

abdalla-elmahdy committed:
Auto stash before revert of "Resolved conflicts"

abdalla-elmahdy committed:
Merge branch 'master' into feature/promotion-contact

Conflicts resolved

abdalla-elmahdy committed:
Merge branch 'master' into feature/promotion-contact

Nilegfx reviewed (approved):
LGTM, Thanks for keeping your commit messages clear and descriptive for other team members! good job 👍
